### DISCUSSION TITLE
https://pyinstaller.org/en/stable/spec-files.html page Never mind

### DISCUSSION THREAD

**Discussion by jschwar313** [pyinstaller/pyinstaller#6860] [Help]
Never mind.

**Comment by bwoodsend** [comment]
No, they can either use `/` or use `\` in a raw string. `/` is recommended since it's cross platform and doesn't lead to surprises if a naive Windows user writes something like `"C:\Users\neil\some\file"`.

**Comment by jschwar313** [comment]
Never mind.

**Reply by jschwar313** [reply]
Never mind.

**Reply by bwoodsend** [reply]
It's exactly that I'm afraid. As much as I'd like to make PyInstaller accessible to newbies, there are much bigger hurdles in the world of making and distributing portable software than Windows filenames (which isn't even really to do with PyInstaller - it's something that should be learnt when learning about strings in Python) so humoring you on this point and adding it to the docs is not going to enable said newbie to produce a working application - rather they're more likely to end up in an even deeper, stickier mess than the one they'd have otherwise landed in before.
